blockchain/vm/contracts.go

```diff
@@ -64,14 +64,14 @@ type PrecompiledContract interface {
 	Run(input []byte, contract *Contract, evm *EVM) ([]byte, error)
 }
 
-// PrecompiledContractsConstantinople contains the default set of pre-compiled Klaytn
-// contracts based on Ethereum Constantinople.
-var PrecompiledContractsConstantinople = map[common.Address]PrecompiledContract{
+// PrecompiledContractsByzantiumCompatible contains the default set of pre-compiled Klaytn
```

Comment:
Why this name is changed? The fields contains constantionple values. Is this correct?

```
	common.BytesToAddress([]byte{6}):  &bn256AddConstantinople{},
	common.BytesToAddress([]byte{7}):  &bn256ScalarMulConstantinople{},
	common.BytesToAddress([]byte{8}):  &bn256PairingConstantinople{},
```